packages/framework/src/Model/Cart/Watcher/CartWatcherFacade.php

```diff
@@ -63,11 +73,12 @@ protected function checkModifiedPrices(Cart $cart)
     {
         $modifiedItems = $this->cartWatcher->getModifiedPriceItemsAndUpdatePrices($cart);
 
+        $messageTemplate = $this->twigEnvironment->createTemplate(
```

Comment:
Having the flash messages in the facade does not seem right to me (and it's the only occurrence). Is it possible to move it into the controller?

Reply:
I thought about it same as you, but in this case it's not easy, I will create the US, are you ok with that?

Reply:
yup, nice :)